tests/test_receivers.py

```diff
@@ -65,3 +67,30 @@ def test_build_cache_with_no_collections(app):
 
         # now the collection 'mycoll' should appear in `record0` of A!
         assert record0['_collections'] == ['mycoll']
+
+
+def test_cache_can_be_imported_from_a_config_variable(app):
+    InvenioCollections(app)
+    with app.app_context():
+        current_app.config['COLLECTIONS_CACHE'] = \
+            'werkzeug.contrib.cache:SimpleCache'
```

Comment:
This is a wrong import and it should fail when you try to add something to cache.